flink-contrib/flink-storm/src/test/java/org/apache/flink/storm/wrappers/StormTupleTest.java

```diff
@@ -595,7 +593,7 @@ public void testGetBinaryByFieldPojoGetter() throws Exception {
 	private <T> StormTuple testGetByField(int arity, int index, T value)
 			throws Exception {
 
-		assert (index < arity);
+		Assert.assertTrue(index < arity);
 
```

Comment:
This is not a assertion for a test result; thus usage of `assert` is correct (checks if test is written properly).